sdk_container/src/third_party/coreos-overlay/coreos-base/coreos/coreos-0.0.1.ebuild

```diff
@@ -104,7 +104,6 @@ RDEPEND="${RDEPEND}
 	app-crypt/gnupg
 	app-crypt/go-tspi
 	app-crypt/tpmpolicy
-	app-editors/vim
```

Comment:
To state what was written elsewhere: We might want to still include an emergency editor in the base image. Something like `nano` is good because it's common, user friendly, and minimal (and many distros have it as default editor).

Reply:
Unrelated comment: I think it would be nice to have mksquashfs and mkfs.erofs available to build sysext images on Flatcar itself, maybe they could also directly go to a tools extension instead of the base image. Not sure if `tools` or optional `extra-tools` would be better - in doubt `extra-tools` to keep the default small?